tests/integration/test.basics.js

```diff
@@ -1106,5 +1106,15 @@ adapters.forEach(function (adapter) {
         });
       });
     }
+
+    if (typeof process !== 'undefined' && !process.browser) {
+      it('#5471 PouchDB.plugin() should throw error if passed wrong type or empty object', function () {
+        PouchDB.plugin('pouchdb-adapter-memory').then(function () {
```

Comment:
the `assert` code did not work (`assert` is not defined? Also do not see `assert` used anywhere else), but a try/catch did work.